Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -1372,6 +1372,28 @@ public void ClearConnectId(ProductUserId userId)
                 }
             }
 
+            //-------------------------------------------------------------------------
+            /// <summary>
+            /// Clears the stored token for persistent login
+            /// </summary>
+            public void RemovePersistentToken()
+            {
+                var authInterface = EOSManager.Instance.GetEOSPlatformInterface().GetAuthInterface();
+                var options = new Epic.OnlineServices.Auth.DeletePersistentAuthOptions();
+
+                authInterface.DeletePersistentAuth(ref options, null, (ref DeletePersistentAuthCallbackInfo deletePersistentAuthCallbackInfo) =>
```

Comment:
I'm so used to C++. Can you remind me what the `ref` keyword does in this context?

Reply:
`ref` is a lot like passing a value by address in C++: it passed it by reference/address instead of by value.